### ISSUE TITLE
desktop browser navigation broken from Smart Dollar back to dashboard

### ISSUE THREAD

**Issue by dcvezzani** [dcvezzani/welfie#1]
I’ve pushed out your changes to https://welfie.co; it’s really looking good.  I did stumble across an issue with browser navigation.  Tell me what you think about this… 

https://cl.ly/2Z3Z0g3b0y1T/Screen%20Recording%202018-02-24%20at%2010.24%20AM.gif

The issue only seems to manifest on my desktop Chrome browser.  It seems to work as expected on my iPhone (iOS Safari).

**Comment by riceboyler** [comment]
Yeah, the problem here is that we're using Browser Router, and we don't have a server router in place to handle routing before it gets to React. Let me look up the best fix for this.
